pkg/compiler/codegen.go

```diff
@@ -1268,6 +1268,19 @@ func (c *codegen) convertBuiltin(expr *ast.CallExpr) {
 	}
 
 	switch name {
+	case "make":
+		typ := c.typeOf(expr.Args[0])
+		switch {
+		case isMap(typ):
+			emit.Opcode(c.prog.BinWriter, opcode.NEWMAP)
+		case isByteSlice(typ):
+			ast.Walk(c, expr.Args[1])
```

Comment:
What about the second argument?

Reply:
It is a capacity, we ignore it. It will lead to differences between NeoVM and Go code, but there is not much we can do, as `MEMCPY` checks arguments `len`s anyway.

Reply:
We can emit explicit error.

Reply:
I think we should to set proper expectations on users side.

Reply:
Fixed.